jaxrs-api/src/main/java/jakarta/ws/rs/client/RxInvoker.java

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2015, 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025 Contributors to the Eclipse Foundation
```

Comment:
Same here. Stopping review, as apparently lots of files have the same problem.

Reply:
Reverted (c) changes.